scm-ui/ui-components/src/SyntaxHighlighter.tsx

```diff
@@ -74,7 +92,10 @@ const SyntaxHighlighter: FC<Props> = ({ language = defaultLanguage, showLineNumb
       >
         {value}
       </ReactSyntaxHighlighter>
-    </div>
+      <TopRightButton title={t("syntaxHighlighting.copyButton")} onClick={copy}>
+        <i className={copied ? "fa fa-clipboard-check" : "fa fa-clipboard"} />
+      </TopRightButton>
```

Comment:
This new button appears underneath the "markdown-switch-button" and is nearly invisible. This is not what you intended, is it?

Reply:
Ah, ok, this is ... hmpf. Do you have a suggestion?